src/umpire/Umpire.cpp

```diff
@@ -209,6 +209,8 @@ umpire::MemoryResourceTraits get_default_resource_traits(const std::string& name
 void* find_pointer_from_name(Allocator allocator, const std::string& name)
 {
   void* ptr{nullptr};
+  // Using unused attribute to silence warning from rocm compiler
+  UMPIRE_USE_VAR(allocator);
```

Comment:
If you move this down to the `#else` block where the `UMPIRE_USE_VAR(name);` is used, does it compile? (I think it should)